### ISSUE TITLE
accountExpires attribute returned without formatting.  

### ISSUE THREAD

**Issue by w-p** [cannatag/ldap3#105]
I have only seen this issue with the __accountExpires__ attribute.  Upon returning from a search, it is not being converted to a __datetime__ object like other such attributes (lastLogon, badPasswordTime, etc.).

Clearly other values are being formatted with __format_ad_timestamp()__ but __accountExpires__ is consistently returned as a binary string.

**Comment by cannatag** [comment]
Hi Will, what is the syntax of this attribute in the schema? Are you using the get_info=SCHEMA parameter in the server object? If the accountExpires has a custom format it can be added to the standard formatter of ldap3.

**Comment by w-p** [comment]
The accountExpires attribute is an AD timestamp like lastLogon and badPasswordTime.

accountExpires
attribute: https://msdn.microsoft.com/en-us/library/ms675098(v=vs.85).aspx
schema: https://msdn.microsoft.com/en-us/library/Cc221084.aspx

badPasswordTime
attribute: https://msdn.microsoft.com/en-us/library/ms675243(v=vs.85).aspx
schema: https://msdn.microsoft.com/en-us/library/Cc221229.aspx

Both of these attributes have the same attributeSyntax and omSyntax values however, while badPasswordTime is sent through the formatter, accountExpires is not.

**Comment by w-p** [comment]
Apologies, the issue is apparently not that the attribute itself is not being formatted but rather that the value presented by AD is not being accounted for.  In the case of __accountExpires__ the AD timestamp uses the value __9223372036854775807__ to indicate that the account will never expire.

I don't think there is an infinity representation of a __datetime__ object in Python so perhaps returning __float('inf')__ in this particular case would make sense?  This works nicely for JSON representation.

**Comment by cannatag** [comment]
HI Will, that value is used to represent the highest value that can be stored in a 64 bit signed integer ( 2^63 − 1 or  7FFF FFFF FFFF FFFF in hex) and in this case should mean the highest date that can be stored in the accountExpires attribute.

To be consistent with the value returned by format_ad_timestamp() I prefer:

```
        if raw_value == b'9223372036854775807':  # max value to be stored in a 64 bit signed int
            return datetime.max  # returns datetime.datetime(9999, 12, 31, 23, 59, 59, 999999)
```

This return the highest date managed by the python datetime module and should not break existing code.

**Comment by w-p** [comment]
I was just about to revise my previous assertion.  You are correct.  Staying standard is the right way to go.  Thanks.

**Comment by cannatag** [comment]
Will release it in the next release.

bye,
Giovanni

**Comment by cannatag** [comment]
Fixed in 0.9.9
